src/appleseed-max-impl/appleseedinteractive/appleseedinteractive.cpp

```diff
@@ -319,10 +319,13 @@ void AppleseedInteractiveRender::BeginSession()
         get_view_params_from_viewport(view_params, *GetViewExp(), m_time);
 
     Renderer* curr_renderer = GetCOREInterface()->GetCurrentRenderer(false);
+    // This is not production appleseed renderer, this is completetly independent renderer object.
```

Comment:
Just wanted to have a reminder here because I've spent some time to figure out which object is accessed here. I can remove it. Or add "Reminder:"

Reply:
Sure, but what does the comment mean? What do you mean by "production appleseed renderer"?

Reply:
I thought it's confusing because I started with non-interactive renderer (production) and then switched to interactive. Then I saw that line and I thought it tries to access production renderer to take it's settings. But in reality it's accessing interactive renderer which is also `AppleseedRender` object. That object later produces `AppleseedInteractiveRender`.
It is all very confusing when you forgot about it. Probably it's clear enough if I think carefully. I can remove that comment, it doesn't make things clearer. I'll add something better next time it confuses me.

Reply:
So line `RendererSettings renderer_settings = appleseed_renderer->get_renderer_settings();` basically accesses default settings. But it looks like it tries to copy settings from somewhere, right?

Reply:
Regarding production vs interactive renderer: I understand now. It makes a lot of sense to explain this in a comment.

Reply:
Regarding renderer settings: I'm not sure to understand the question, but 

    RendererSettings renderer_settings = appleseed_renderer->get_renderer_settings();

clearly _copies_ the renderer settings into the local `renderer_settings` variable.